View/Templates/Registration.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="../Assets/CSS/style.css">
+    </head>
+    <body>
+
+        <div class="rgstrdiv">
+        <form action="Registration.html">
+            <h1> Register </h1>
+            
+            <label class = "labels">FirstName: </label><br/>
+            <input type="text" placeholder="FirstName" name="firstname" class="fields" required> <br/><br/>
```

Comment:
Don't use `br` to give space between 2 fields. Use the `div` tag and add a margin between the `div`